src/lib/y2storage/dialogs/guided_setup/select_filesystem/volume_widget.rb

```diff
@@ -147,6 +163,28 @@ def header_for_lvm
             end
           end
 
+          # @see #header_term
+          def header_for_vg
+            case volume.mount_point
+            when "/"
+              # TRANSLATORS: "Volume" refers to an LVM logical volume.
+              _("Settings for the Root Volume Group")
+            when "/home"
+              # TRANSLATORS: "Volume" refers to an LVM logical volume.
+              _("Settings for the Home Volume Group")
+            when "swap"
+              # TRANSLATORS: "Volume" refers to an LVM logical volume.
+              _("Settings for Swap Volume Group")
+            when nil
+              # TRANSLATORS: "Volume" refers to an LVM logical volume and
+              # "Additional" implies it will be created but not mounted
+              _("Settings for Additional Volume Group")
+            else
+              # TRANSLATORS: "Volume" refers to a LVM logical volume. %s is a mount point.
+              _("Settings for the %s Volume Group") % volume.mount_point
```

Comment:
Yes, I know. But I think that we also should keep a certain grade of consistency, and there are already similar translations in the file using the other way. Should we to adapt this new line or keep it as it is?

Reply:
Ideally, I would fix all translations in that file to use "format". If there are many, I would only change this new.

Reply:
There are a few more. Two or three, I don't remember. But we had problems with translations in the past because of this (I can't remember which exact PR, but I remember that @ancorgs looked at me and said: I told you! :stuck_out_tongue_winking_eye:)